lib/v2/hameln/chapter.js

```diff
@@ -0,0 +1,58 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+    const limit = parseInt(ctx.query.limit) || 5;
+    const link = `https://syosetu.org/novel/${id}`;
+    const $ = cheerio.load(await get(link));
+
+    const title = $('span[itemprop="name"]').text();
+    const description = $('div.ss:nth-child(2)').text();
+
+    const chapter_list = $('tr[bgcolor]')
+        .map((_, chapter) => {
+            const $_chapter = $(chapter);
+            const chapter_link = $_chapter.find('a');
+            return {
+                title: chapter_link.text(),
+                link: chapter_link.attr('href'),
+                pubDate: parseDate($_chapter.find('nobr').text(), 'YYYYMMDD HH:mm'),
```

Comment:
```suggestion
                pubDate: timezone(parseDate($_chapter.find('nobr').text(), 'YYYYMMDD HH:mm'), +9),
```
docs: https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei-timezone